src/js/App/Sidenav/SectionNav.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { NavGroup } from '@patternfly/react-core/dist/js/components/Nav/NavGroup';
+import ExpandableNav from './ExpandableNav';
+import './SectionNav.scss';
+
+const SectionNav = ({ items, title, id, onClick, ...props }) => {
+  if (items?.length > 0) {
+    return (
+      <React.Fragment>
+        <NavGroup className="ins-c-section-nav" id={id} title={title.toUpperCase()}>
+          {items.map((item, key) => (
+            <ExpandableNav
+              key={item.id || key}
+              {...props}
+              {...item}
+              onClick={(event, subItem) => (item.subItems ? onClick(event, subItem, item) : onClick(event, item))}
+            />
+          ))}
+        </NavGroup>
+      </React.Fragment>
+    );
+  }
+  const item = { id: id, title: title, props };
```

Comment:
No need to assign to same name and props should be spread.
```suggestion
  const item = { id, title, ...props };
```